Makefile

```
@@ -114,10 +114,9 @@ lint:
 
 .PHONY: lint-fix
 lint-fix:
-	python3 -c "import black" > /dev/null 2>&1 || python3 -m pip install black
-	python3 -m black .
 	python3 -c "import ruff" > /dev/null 2>&1 || python3 -m pip install ruff
-	python3 -m ruff --fix .
+	python3 -m ruff check --fix .
+	python3 -m ruff format .
```

```suggestion
	python3 -m ruff check --fix
	python3 -m ruff format
```
See https://github.com/astral-sh/ruff/pull/10217